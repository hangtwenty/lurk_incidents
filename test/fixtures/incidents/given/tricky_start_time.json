{
    "backfilled": false,
    "created_at": "2016-09-08T09:35:23.253-04:00",
    "id": "wjkdk2kylkn3",
    "impact": "none",
    "impact_override": null,
    "incident_updates": [
        {
            "affected_components": [
                {
                    "name": "No components were affected by this update."
                }
            ],
            "body": "The issue has been addressed. Checkout functionality has returned to normal",
            "created_at": "2016-09-08T09:38:07.466-04:00",
            "display_at": "2016-09-03T17:28:00.000-04:00",
            "id": "8jph7kn575td",
            "incident_id": "wjkdk2kylkn3",
            "status": "resolved",
            "twitter_updated_at": null,
            "updated_at": "2016-09-08T09:38:26.362-04:00",
            "wants_twitter_update": false
        },
        {
            "affected_components": [
                {
                    "name": "No components were affected by this update."
                }
            ],
            "body": "We\u2019ve identified a problem with checkout and are working on a solution. We\u2019ll have this fixed soon.",
            "created_at": "2016-09-08T09:35:23.791-04:00",
            "display_at": "2016-09-03T17:25:00.000-04:00",
            "id": "77r8bm3lwx3k",
            "incident_id": "wjkdk2kylkn3",
            "status": "identified",
            "twitter_updated_at": null,
            "updated_at": "2016-09-08T09:36:42.746-04:00",
            "wants_twitter_update": false
        }
    ],
    "monitoring_at": null,
    "name": "Unexpected problem, Errors returned when attempting to checkout",
    "page_id": "d33g96wd23dd",
    "postmortem_body": null,
    "postmortem_body_last_updated_at": null,
    "postmortem_ignored": false,
    "postmortem_notified_subscribers": false,
    "postmortem_notified_twitter": false,
    "postmortem_published_at": null,
    "resolved_at": "2016-09-03T17:28:00.000-04:00",
    "scheduled_auto_completed": false,
    "scheduled_auto_in_progress": false,
    "scheduled_for": null,
    "scheduled_remind_prior": false,
    "scheduled_reminded_at": null,
    "scheduled_until": null,
    "shortlink": "http://stspg.io/3Qw3",
    "status": "resolved",
    "updated_at": "2016-09-08T09:38:26.397-04:00"
}
