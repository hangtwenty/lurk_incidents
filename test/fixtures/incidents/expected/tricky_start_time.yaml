---
# if you look at the 'given' in this case, you'll see the 'started_at' comes from a nested field...
# it comes from the 'display_at' for the first update.
# it seems this is the kinda case where an incident is added retroactively, but not all fields plugged in.
started_at: "2016-09-03T17:25:00.000-04:00"
ended_at: "2016-09-03T17:28:00.000-04:00"
